event/events.go

```diff
@@ -118,5 +135,25 @@ func GenerateSubId() (string, error) {
 	}
 	rStr := hex.EncodeToString(b)
 	return strings.ToUpper(rStr), nil
+}
 
+func mapToOurEventData(eventData anyEventData) (txs.EventData, error) {
+	// While we depend on go-events in the way we do, we don't have much choice
```

Comment:
Can we not define a new `txs` event type (or even define it at `/consensus`) that wraps around the go-events such that we swallow these event types, rather than this naked switch?  -- I'm not radically opposed if this resolves the bug in the short-term and we can add a comment of following discussion.